content/posts/fleek-network-running-a-node-in-a-docker-container.md

```diff
@@ -191,30 +245,35 @@ A container is what's originated from the image we discussed [earlier](#image),
 
-### Run the container from official nightly image
+### Run the container from the official nightly image
 
+<!-- 
 Building a Docker image requires some effort and some of our users might find easier to pull our nightly image for quick access to Ursa, which does not require them to build from source.
 
-For that reason, we provide you a quick demonstration on how to pull and run the Docker container! If you need more detailed information, then you'll be happier to follow the [run the Docker container](#run-the-docker-container).
+For that reason, we provide you a quick demonstration on how to pull the image and run the Docker container quickly! If you need more detailed information, then you'll be happier to follow the [run the Docker container](#run-the-docker-container). -->
 
-> The official `fleek-network/ursa:nightly` image is currently hosted in Github's container registry and updated every night, you can `docker pull` the image and run it locally. Beware that the version is built at a particular time of the day, so if you are looking for a custom image then you're better off learning how to build one yourself. The extended guide provides you all the information you need, but we also have a guide on how to update images that you may be interested in reading [here](https://blog.fleek.co/posts/fleek-network-how-to-get-the-latest-updates-for-ursa-cli-from-the-source-repository).
+The Fleek Network Ursa image is built nightly, and some of our users might find handy to pull it instead of building it for quick access to a running node!
 
-Looking for the `fleek-network/ursa:nightly`? Find the Github package page [here](https://github.com/fleek-network/ursa/pkgs/container/ursa).
+> The official [fleek-network/ursa:nightly](https://github.com/fleek-network/ursa/pkgs/container/ursa) image is currently hosted in Github's container registry and updated every night, you can `docker pull` the image and run it locally. Beware that the version is built at a particular time of the day, so if you are looking for a custom image then you're better off learning how to build one yourself. The extended guide provides you all the information you need, but we also have a guide on how to update images that you may be interested in reading [here](https://blog.fleek.co/posts/
 
 Firstly, start by running the `docker pull`, as follows:
 
 ```sh
 docker pull ghcr.io/fleek-network/ursa:nightly
 ```
 
+⚠️ Important: The Github container registry is private, you need a Github account and private token to login via the CLI to be able to pull, find the instructions [here](https://ghcr.io/).
+
 Once the Docker image is downloaded completely, you can run a container based on the image:
 
 ```sh
-docker run -p 4069:4069 -p 4070:4070 -p 6009:6009 -p 8070:8070 --name ursa-cli -it ghcr.io/fleek-network/ursa:nightly
+docker run -p 4069:4069 -p 4070:4070 -p 6009:6009 -p 8070:8070 -v $HOME/.ursa/:/root/.ursa/:rw --name ursa-cli -it ghcr.io/fleek-network/ursa:nightly
```

Comment:
```suggestion
docker run -p 4069:4069 -p 6009:6009 -v $HOME/.ursa/:/root/.ursa/:rw --name ursa-cli -it ghcr.io/fleek-network/ursa:nightly
```

With fleek-network/ursa#233 can remove index provider port, and unique metrics port has since been removed fleek-network/ursa#81

Reply:
Sure! Regards the name, the name you are seeing there is for the container, not the image. Where `ursa-cli` is the default name we use in the makefile and previous guides.